crates/bevy_text/Cargo.toml

```diff
@@ -26,7 +25,9 @@ bevy_window = { path = "../bevy_window", version = "0.12.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.12.0" }
 
 # other
-ab_glyph = "0.2.6"
-glyph_brush_layout = "0.2.1"
+anyhow = "1.0.4"
```

Comment:
Doesn't seem like anyhow is used in this PR. I'm guessing this line is from before #10003
```suggestion
```